segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/DurableLog.java

```diff
@@ -221,7 +220,7 @@ private boolean performRecovery() {
             RecoveryProcessor p = new RecoveryProcessor(this.metadata, this.durableDataLog, this.memoryStateUpdater);
             int recoveredItemCount = p.performRecovery();
             this.operationProcessor.getMetrics().operationsCompleted(recoveredItemCount, timer.getElapsed());
-            SegmentStoreMetrics.reportOperationLogSize(recoveredItemCount, this.getId());
+            this.operationProcessor.getMetrics().reportOperationLogSize(recoveredItemCount, this.getId());
```

Comment:
If this is returning a `SegmentStoreMetrics.OperationProcessor` object, why `reportOperationLogSize` is defined as `static`?

Reply:
will correct this(had just moved the method)